asset/css/controls.less

```diff
@@ -30,3 +30,54 @@
   }
 }
 
+
+// Style
+
+.control-button {
+  .appearance("none");
+  background: none;
+  border: none;
+  color: @icinga-blue;
+  .rounded-corners();
+
+
+  &:hover, &:focus, &.active {
+    background-color: @gray-lighter;
+  }
+
+  i.icon:before {
+    color: inherit;
+  }
+}
+
+// Layout
+
+.control-button {
+  padding: 0 .5em;
+
+  &:last-child {
+    margin-right: -.5em !important;
```

Comment:
No. Two times. `!important` is just lazy, and why not `padding-right: 0`?

Reply:
`padding-right: 0` will make the button narrower. This doesn't harm in the first place. But the click target will then be smaller and it'll become obvious, when the button is hovered.

Reply:
[This](https://github.com/Icinga/ipl-web/pull/21/commits/cca515bc239a6cafdb095b979ad07a0a0cf9215a#) would be the alternative then.